src/pages/deployments/configure-deployment/ConfigureDeployment.jsx

```diff
@@ -244,6 +287,60 @@ const ConfigureDeployment = props => {
     }
   ]
 
+  const scalersColumn = [
+    {
+      title: 'Name',
+      dataIndex: 'name'
+    },
+    {
+      title: 'Scaler',
+      dataIndex: 'type',
+      render: (_, { type }) => {
+        switch(type){
+          case 'requests-per-second':
+            return capitalizeFirstCharacter(type).replaceAll('-', ' ');
```

Comment:
Since you are using switch case, you can directly return the values `Request per second`, `Active requests`, `CPU`, `RAM` in their respective cases